Expecto/Expecto.fs

```diff
@@ -1565,4 +1564,36 @@ module Accuracy =
   let high =
     {absolute=1e-10; relative=1e-7}
   let veryHigh =
-    {absolute=1e-12; relative=1e-9}
\ No newline at end of file
+    {absolute=1e-12; relative=1e-9}
+
+namespace Expecto.CSharp
+open System.Runtime.CompilerServices
+
+[<AutoOpen; Extension>]
+module Runner =
+  open Expecto
+  open Expecto.Impl
+  open System.Collections.Generic
+  open System.Threading.Tasks
+
+  type Async with
+    static member AwaitTask(t : Task) = Async.AwaitIAsyncResult(t, -1) |> Async.Ignore
```

Comment:
There is a native implementation in F# 4.0 if we can upgrade the FSharp.Core dependency from 3.1.2.5.  Would you be open to this?